luda-editor/luda-editor-client/src/app/types/router_context.rs

```diff
@@ -0,0 +1,7 @@
+use luda_editor_rpc::Socket;
+use namui::Wh;
+
+pub struct RouterContext {
+    pub screen_size: Wh<f32>,
```

Comment:
I think it is not a context of router, but app's context. Would you rename it as `AppContext`?

Reply:
Done with fa990a9